src/ondisk.rs

```diff
@@ -8294,20 +8816,70 @@ make_token_accessors! {
         FchConsoleOutBasicEnable(default 0, id 0xa0903f98) | pub get u8 : pub set u8, // Rome (Obsolete)
         FchConsoleOutSerialPort(default 0, id 0xfff9_f34d) | pub get FchConsoleSerialPort : pub set FchConsoleSerialPort,
+        FchConsoleOutSerialPortIoBase(default 0, id 0x95dc_6839) | pub get FchConsoleOutSerialPortIoBase : pub set FchConsoleOutSerialPortIoBase,
         FchSmbusSpeed(default 42, id 0x2447_3329) | pub get FchSmbusSpeed : pub set FchSmbusSpeed,
         FchConsoleOutSuperIoType(default 0, id 0x5c8d_6e82) | pub get FchConsoleOutSuperIoType : pub set FchConsoleOutSuperIoType, // init mode
+        FchIc3TransferSpeed(default 2, id 0x1ed8_fae5) | pub get FchIc3TransferSpeed : pub set FchIc3TransferSpeed,
+        FchI2cI3cController0Mode(default 0, id 0xb663_3a33) | pub get FchI2cI3cControllerMode : pub set FchI2cI3cControllerMode,
+        FchI2cI3cController1Mode(default 0, id 0xd2c7_be74) | pub get FchI2cI3cControllerMode : pub set FchI2cI3cControllerMode,
+        FchI2cI3cController2Mode(default 0, id 0x9ec3_f26b) | pub get FchI2cI3cControllerMode : pub set FchI2cI3cControllerMode,
+        FchI2cI3cController3Mode(default 0, id 0x83a0_b3d6) | pub get FchI2cI3cControllerMode : pub set FchI2cI3cControllerMode,
+        FchI2cI3cController4Mode(default 0, id 0xd7ac_3048) | pub get FchI2cI3cControllerMode : pub set FchI2cI3cControllerMode,
+        FchI2cI3cController5Mode(default 0, id 0xebc3_3806) | pub get FchI2cI3cControllerMode : pub set FchI2cI3cControllerMode,
+        #[cfg_attr(feature = "serde-hex", serde(serialize_with = "SerHex::<StrictPfx>::serialize", deserialize_with = "SerHex::<StrictPfx>::deserialize"))]
+        FchIc3PushPullHighCount(default 0x08, id 0xe180_bc26) | pub get u8 : pub set u8,
+
+        FchI2cI3cSmbusSelect0(default 1, id 0x811f_7fbd) | pub get FchI2cI3cSmbusSelect : pub set FchI2cI3cSmbusSelect,
+        FchI2cI3cSmbusSelect1(default 1, id 0x32bd_7f95) | pub get FchI2cI3cSmbusSelect : pub set FchI2cI3cSmbusSelect,
+        FchI2cI3cSmbusSelect2(default 1, id 0x8e27_5326) | pub get FchI2cI3cSmbusSelect : pub set FchI2cI3cSmbusSelect,
+        FchI2cI3cSmbusSelect3(default 1, id 0xed13_9de8) | pub get FchI2cI3cSmbusSelect : pub set FchI2cI3cSmbusSelect,
+
+        // TODO: Replaced at some point with APCB_TOKEN_UID_I2C_4 / APCB_TOKEN_UID_I2C_5 ?
+        FchI2cI3cSmbusSelect4(default 1, id 0x2158_efca) | pub get FchI2cI3cSmbusSelect : pub set FchI2cI3cSmbusSelect,
+        FchI2cI3cSmbusSelect5(default 1, id 0x5caa_a37b) | pub get FchI2cI3cSmbusSelect : pub set FchI2cI3cSmbusSelect,
+
+        // TODO: Maybe should be called `FchI2cSdaRxHold` but that's already a WordToken
+        FchIcSdaRxHold(default 0, id 0xa4ba_c3d5) | pub get u8: pub set u8,
+        // Same as `FchI2cSdaHoldOverrideMode` but that's already a WordToken
+        FchI2cSdaHoldOverrideMode2(default 0, id 0x545d_7662) | pub get FchI2cSdaHoldOverrideMode : pub set FchI2cSdaHoldOverrideMode,
```

Comment:
If we wanted to we could totally remove the direct token interface that takes all the tokens, and for each of them, generates an accessor at the top level with just the token name.

I don't think we have any consumers of that interface anyway.

Not sure what AMD's intention here is. Is it so everyone requires hungarian notation to make sense of the things? ;P